src/mame/video/segasaturn_vdp1.h

```diff
@@ -0,0 +1,286 @@
+// license:LGPL-2.1+
+// copyright-holders:David Haywood, Angelo Salese, Olivier Galibert, Mariusz Wojcieszek, R. Belmont
+/**************************************************************************************************
+
+Sega Saturn / ST-V - VDP1
+
+**************************************************************************************************/
+
+#ifndef MAME_VIDEO_SATURN_VDP1_H
+#define MAME_VIDEO_SATURN_VDP1_H
+
+#pragma once
+
+
+#define RGB_R(_color)   (_color & 0x1f)
+#define RGB_G(_color)   ((_color >> 5) & 0x1f)
+#define RGB_B(_color)   ((_color >> 10) & 0x1f)
```

Comment:
Can you change these to `constexpr` functions, and preferably move them out of the global scope?  They have very generic names, which can be problematic.